proto/vtctldata.proto

```diff
@@ -194,6 +195,34 @@ message ApplyRoutingRulesRequest {
 message ApplyRoutingRulesResponse {
 }
 
+message ApplySchemaRequest {
```

Comment:
protobuf changes need an extra build step.
From slack:

> For folks that update the .proto files with `make proto`, it would be very appreciated if, for now, you could also run the `make vtadmin_web_proto_types` command. This ensures that VTAdmin's generated TypeScript files stay in sync with the proto files.
I know that remembering things is the worst :) so this is only necessary until we can do this automagically in CI. (https://github.com/vitessio/vitess/issues/9571 is the ticket for that work.)
Note that running `make vtadmin_web_proto_types` requires an install of node LTS (>= 16.13). If your node version is too old, you may run into an error like this one. Node v16.13.x (LTS) can be installed via https://nodejs.org, or alternatively can be managed with something like nvm.

Reply:
Done!

Reply:
Thank you both, it is appreciated. :)